src/workloads/execution/FilterWithLogicalExpressions.yml

```diff
@@ -0,0 +1,162 @@
+SchemaVersion: 2018-07-01
```

Comment:
This only covers expressions, which makes sense. Is your intention to add workloads to cover complex `MatchExpression`s as a separate task? That would be fine with me, I just want to make sure I understand your plan.

Reply:
Initially, I was thinking it will be a separate workload in a separate PR. But now I am thinking that we should just go ahead and add it here, since the workloads themselves are quite compact. Added find alternative for each of the queries.

Reply:
Sounds good to me! Resolving.